shared_model/validators/validators_common.hpp

```diff
@@ -17,13 +17,19 @@ namespace shared_model {
      */
     struct ValidatorsConfig {
       ValidatorsConfig(uint64_t max_batch_size,
-                       bool partial_ordered_batches_are_valid = false);
+                       bool partial_ordered_batches_are_valid = false,
+                       bool duplicate_tolerance = true);
       /// Maximum allowed amount of transactions within a batch
       const uint64_t max_batch_size;
 
       /// Batch meta can contain more hashes of batch transactions than it
       /// actually has. Used for block validation
       const bool partial_ordered_batches_are_valid;
+
+      /// Flag for choosing strategy of working with duplicates of transaction
+      /// if true, then duplicates will be ignored
+      /// if false, then collection will be invalid
+      const bool duplicate_tolerance;
```

Comment:
Sorry but I see poor wording and unclear meaning
```diff
-      /// Flag for choosing strategy of working with duplicates of transaction
-      /// if true, then duplicates will be ignored
-      /// if false, then collection will be invalid
-      const bool duplicate_tolerance;
+    /**
+     * Defines whether a transactions collection, with duplicated transactions, valid or not.
+     * Used in:
+     * - TransactionBatchValidator (ListTorii)
+     * - MST & OrderingGate & OrderingService
+     * - BlockLoader
+     */
+    const bool txs_duplicates_are_allowed;
```

Reply:
Fixed